public/index.html

```diff
@@ -16,7 +16,8 @@
 		<script type="text/javascript">
 		window.onload = function() {
 			if (document.getElementById('app')) {
-				document.writeln("Sorry, this website requires a modern browser. Please update your browser or use Google Chrome or Mozilla Firefox as alternative.");
+				document.writeln("<h1>Broken openEO Web Editor</h1>");
```

Comment:
```suggestion
				document.writeln("<h1>openEO Web Editor unable to load</h1>");
```

Reply:
```suggestion
				document.writeln("<h1> openEO Web Editor failed to load</h1>");
```
make it more a proper sentence?